dev_guide/new_app.adoc

```diff
@@ -12,100 +12,82 @@ toc::[]
 
 == Overview
 
-You can create a new {product-title} application using the
-xref:../architecture/infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console] or
-by running the `oc new-app` command xref:using-the-cli[from the CLI].
-{product-title} creates a new application by specifying
-xref:specifying-source-code[source code], xref:specifying-an-image[images], or
-xref:specifying-a-template[templates]. The `new-app` command looks for images
-on the local Docker installation (if available), in a
-xref:../architecture/infrastructure_components/image_registry.adoc#architecture-infrastructure-components-image-registry[Docker
-registry], or an {product-title}
-xref:../architecture/core_concepts/builds_and_image_streams.adoc#image-streams[image
-stream].
-
-If you specify source code, `new-app` attempts to construct a
-xref:builds.adoc#defining-a-buildconfig[build configuration] that builds your
-source into a new application
-xref:../architecture/core_concepts/builds_and_image_streams.adoc#image-streams[image].
-It also constructs a
-xref:../architecture/core_concepts/deployments.adoc#deployments-and-deployment-configurations[deployment
-configuration] that deploys that new image, and a
-xref:../architecture/core_concepts/pods_and_services.adoc#services[service] to
-provide load balanced access to the deployment that is running your image.
-
-[NOTE]
-====
-If you xref:specifying-source-code[specify source code], you may need to
-xref:builds.adoc#starting-a-build[run a build] with `oc start-build` after the
-application is created.
-====
+You can create a new {product-title} application from source code, images, or templates by using the CLI or the web
+console.
```

Comment:
@adellape basically, yeah.

There's a bit of a story to do with this PR. I'll bring it up in the meeting tonight (tomorrow, your time).